Cython/Utility/Builtins.c

```diff
@@ -148,7 +148,7 @@ static CYTHON_INLINE PyObject *__Pyx_GetAttr3(PyObject *, PyObject *, PyObject *
 //@requires: Exceptions.c::PyErrFetchRestore
 //@requires: Exceptions.c::PyErrExceptionMatches
 
-#if __PYX_LIMITED_VERSION_HEX < 0x030d00A1
+#if __PYX_LIMITED_VERSION_HEX < 0x030D0000
```

Comment:
I chose 'd' here because 'D' looks too close to '0'.

Reply:
Similarly, `B` looks pretty close to `0`, too, in some fonts.

Reply:
No, no my friend... the real problem is you eyes are aging... :rofl: 

OK, now back to being serious: You want me to revert all these uppercase changes? Or maybe make everything lowercase, at least when `__PYX_LIMITED_VERSION_HEX` is involved? CPython headers are not consistent about this admittedly silly matter.

Reply:
Uppercase changes reverted.